maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gittest/src/main/java/org/apache/maven/scm/provider/git/GitScmTestUtils.java

```diff
@@ -88,6 +92,28 @@ public static void deleteAllDirectories(File startDirectory, String pattern) thr
         }
     }
 
+    /**
+     * Creates a new file below a new temporary directory and copies the content of a classpath resource into it.
+     * The caller is responsible for deleting the temporary directory afterwards.
+     * @param clazz
+     * @param resourceName (relative to {@code clazz})
+     * @param fileName
+     * @return the newly created file
+     * @throws IOException
+     */
+    public static Path createTempFileFromClasspathResource(Class<?> clazz, String resourceName, String fileName)
```

Comment:
is this is only used in one place, it probably belongs there instead

Reply:
This will be useful for other tests.

Reply:
Done